homeassistant/components/sms/notify.py

```diff
@@ -42,7 +42,15 @@ async def async_send_message(self, message="", **kwargs):
             _LOGGER.error("No target number specified, cannot send message")
             return
 
-        is_unicode = kwargs.get(CONF_UNICODE, True)
+        extended_data = kwargs.get(ATTR_DATA, {})
+        _LOGGER.debug("Extended data:%s", extended_data)
+
+        if extended_data is None:
+            is_unicode = True
+        else:
+            is_unicode = extended_data.get(CONF_UNICODE, True)
+        _LOGGER.debug("Unicode:%s", is_unicode)
```

Comment:
Please remove this debug log, its redundant and part of the extended_data you logged a few lines above.

Reply:
Done